packages/message-manager/src/AbstractMessageManager.ts

```diff
@@ -370,6 +371,23 @@ export abstract class AbstractMessageManager<
    */
   abstract prepMessageForSigning(messageParams: PM): Promise<P>;
 
+  /**
+   * Creates a new Message with an 'unapproved' status using the passed messageParams.
+   * this.addMessage is called to add the new Message to this.messages, and to save the
+   * unapproved Messages.
+   *
+   * @param messageParams - The params for the personal_sign call to be made after the message
```

Comment:
We can remove this old reference to `personal_sign`.

Reply:
Done